ci-operator/config/openshift/microshift/openshift-microshift-main.yaml

```diff
@@ -131,7 +131,7 @@ tests:
       DRY_RUN: "y"
     workflow: openshift-microshift-rebase
 - always_run: false
-  as: e2e-openshift-conformance-reduced
+  as: e2e-openshift-conformance-reduced-rhel9.2
```

Comment:
Should we say `rhel-eus` instead of including the version number? Or do we want the history of 9.2 and 9.4 to be separate?

Reply:
That sounds fine assuming with 9.4 you do not intend to also keep testing 9.2.

Reply:
Renamed jobs to rhel-eus.